docs/en/interfaces/predefined_http.md

```diff
@@ -0,0 +1,216 @@
+# Predefined HTTP Interface {#predefined_http_interface}
+
+ClickHouse already supports specific queries through the [HTTP interface](../../interfaces/HTTP/http.md) . For example, you can write data to a table as follows:
+ 
+```bash
+$ echo '(4),(5),(6)' | curl 'http://localhost:8123/?query=INSERT%20INTO%20t%20VALUES' --data-binary @-
+```
+
+Now ClickHouse supports Predefined HTTP Interface which can help you more easy integration with third party tools. For example, the following example describes how to by Prometheus with Predefined HTTP Interface integration.
```

Comment:
we shouldn't use any reference to time in documentation (now, already, etc.), here and in other places